my_view_inc.php

```diff
@@ -75,7 +75,9 @@
 
 # @TODO cproensa: make this value configurable
 $t_recent_days = 30;
-$c_filter['recent_mod'] = filter_create_recently_modified( $t_recent_days );
+$t_recent_mod_filter = filter_create_recently_modified( $t_recent_days );
+$t_recent_mod_filter[FILTER_PROPERTY_HIDE_STATUS] = META_FILTER_NONE;
```

Comment:
Why wouldn't this be part of filter_create_recently_modified().  This is the only place that we use this API and should produce the proper filter that can be re-used elsewhere overtime, like in the SOAP API or other places.